feature-libs/asm/styles/components/_customer-list.component.scss

```diff
@@ -412,5 +412,11 @@ $sapFioriContent_LabelColor: #6a6d70;
         box-shadow: none;
       }
     }
+
+    .sort-list {
```

Comment:
Create a new class seems unnecessary, just add it directly
eg:
.ng-select {
    .ng-select-container {
      border: 0;
      .ng-arrow-wrapper {
        .ng-arrow {
          border-color: $sapFioriLinkColor transparent transparent;
        }
      }
    }
    
     @include forVersion(6.1) {
      .ng-dropdown-panel {
        width: auto;
      }
    }
  }